_posts/chausuble/2017-05-22-erin-turtlehack.md

```diff
@@ -0,0 +1,29 @@
+--- 
+layout: post
+author: chausuble
+title: "Erin's Turtlehack"
+---
+
+I thought about this before going to bed one night, and I worked on this over the weekend. 
```

Comment:
Thinking before coding! 🎼 to my ears.  As I said on the first day, the hard work of coding happens away from the computer.